src/engagement_db_to_analysis/engagement_db_to_analysis.py

```diff
@@ -90,34 +77,36 @@ def _get_project_messages_from_engagement_db(analysis_configurations, engagement
                 cache.set_latest_message_timestamp(engagement_db_dataset, latest_message_timestamp)
 
             # Export project engagement_dataset files
-            cache.export_engagement_db_dataset(engagement_db_dataset, messages)
+            cache.set_messages(engagement_db_dataset, messages)
 
     return engagement_db_dataset_messages_map
 
+
 def _convert_messages_to_traced_data(user, messages_map):
     """
     Converts messages dict objects to TracedData objects.
 
     :param user: Identifier of user running the pipeline.
     :type user: str
     :param messages_map: Dict containing messages data.
```

Comment:
engagement_db_dataset_messages_map of engagement_db_dataset to list of messages.

Reply:
Thanks, I see now that this doc was very unclear so have fixed